packages/zendesk-adapter/src/filters/guide_arrange_paths.ts

```diff
@@ -343,6 +343,25 @@ const filterCreator: FilterCreator = () => ({
           parent: parentsById[parentId],
         })
       })
+
+    await awu(guideGrouped[ARTICLE_ATTACHMENT_TYPE_NAME])
+      .forEach(async attachment => {
+        const staticFile: StaticFile = attachment.value.content
```

Comment:
I don't think we need the cast, because of the typeguard in the next line
```suggestion
        const staticFile = attachment.value.content
```